tox.ini

```diff
@@ -5,17 +5,15 @@ max-line-length = 100
 
 [tox]
 envlist = 
-    py{27}-dj{17},
-    flake8
+    python,
+    flake8,
+    functional
 skipsdist = True
 
-[testenv]
-basepython=
-  py27: python2.7
+[testenv:python]
 commands=
-  - cp -n {toxinidir}/config/settings/local_untracked.py.dist {toxinidir}/config/settings/local_untracked.py
+  cp -n {toxinidir}/config/settings/travis.py {toxinidir}/config/settings/local_untracked.py
```

Comment:
I would prefer that the `tox.ini` file not have travis specific stuff if possible so that it can be used as a local test harness.